docs-ref-conceptual/index.yml

```
@@ -17,49 +17,49 @@ metadata:
   ms.devlang: nodejs
   ms.service: multiple
 sections:
-- title: Get started fast 
+- title: Deploy your first Node.js app for free!
   items:
   - type: paragragh
-    html: <p>See how easy it is to get started with Node.js on Azure!
+    html: <p>You can deploy Node.js sample apps for free on Azure App Service</p>
   - type: list
     style: cards
     className: cardsM
     columns: 3
     items:
     - href: https://aka.ms/try-azure-node
-      html: <p>Deploy your first app in a free sandbox environment</p>
+      html: <p>Deploy your first app in a free sandbox environment on Azure App Service</p>
       image:
         src: https://docs.microsoft.com/en-us/media/common/i_get-started.svg
-      title: Get started free!
-- title: Host applications 
+      title: Deploy your first Node.js sample app
+- title: Choose the right hosting for your app
   items:
   - type: paragragh
-    html: <p>Build, deploy, and scale your Node.js applications on Azure
+    html: <p>Azure supports many hosting and application types for Javascript and Node.js developers. Chose your application type below, and we'll guide you how to build, deploy, and scale your application(s) on Azure.</p>
   - type: list
     style: cards
     className: cardsM
     columns: 3
     items:
-    - href: https://code.visualstudio.com/tutorials/static-website/getting-started
+    - title: Static Web Apps, SPAs and Progressive Web Apps
+      html: <p>Deploy static apps built with frameworks like React, Angular, and Vue on Azure Storage</p>
+      href: https://code.visualstudio.com/tutorials/static-website/getting-started
       image:
-        src: https://docs.microsoft.com/media/logos/logo_React.svg
-      title: Static site
-      html: <p>Host your React, Angular, and Vue apps with Azure Storage</p>
-    - href: https://code.visualstudio.com/tutorials/app-service-extension/getting-started
-      html: <p>Deploy Express apps with App Service</p>
+        src: https://docs.microsoft.com/media/logos/logo_React.svg      
+    - title: Full-stack Node.js-powered Web Apps and APIs
+      html: <p>Deploy full-stack node.js web apps and APIs built with frameworks like Express on Azure App Service</p>
+      href: https://code.visualstudio.com/tutorials/app-service-extension/getting-started
       image:
         src: https://docs.microsoft.com/media/logos/logo_vs-code.svg
-      title: Full stack
-    - href: https://code.visualstudio.com/tutorials/docker-extension/getting-started
-      html: <p>Build and deploy docker containers and microservices on Azure</p>
+    - title: Containerized Apps and Microservices with Docker
```

Use sentence capitalization in titles except when the words are part of the proper name of a product.

```suggestion
    - title: Containerized apps and microservices with Docker
```